tutorial/tuto7.py

```diff
@@ -0,0 +1,122 @@
+import os
+from fpdf import FPDF
+from fpdf.enums import OutputIntentSubType
+from fpdf.output import PDFICCProfileObject
+from fpdf import FPDF_VERSION
+from datetime import datetime, timezone
+import pikepdf  # pylint: disable=import-error
+
+
+class PDF(FPDF):
+    """class for my pdf inherited from FPDF"""
+
+    def __init__(self):
+        super().__init__()
+        # import and embed TTF Font to use in text
+        self.add_font(
+            "dejavu-sans", style="", fname="./assets/Fonts/DejaVuSansCondensed.ttf"
+        )
+        self.add_font(
+            "dejavu-sans",
+            style="b",
+            fname="./assets/Fonts/DejaVuSansCondensed-Bold.ttf",
+        )
+        self.add_font(
+            "dejavu-sans",
+            style="i",
+            fname="./assets/Fonts/DejaVuSansCondensed-Oblique.ttf",
+        )
+        self.add_font(
+            "dejavu-sans",
+            style="bi",
+            fname="./assets/Fonts/DejaVuSansCondensed-BoldOblique.ttf",
+        )
+        # set Output Intents
+        with open(os.path.join("assets", "sRGB2014.icc"), "rb") as iccp_file:
+            icc_profile = PDFICCProfileObject(
+                contents=iccp_file.read(), n=3, alternate="DeviceRGB"
+            )
+        self.add_output_intent(
+            OutputIntentSubType.PDFA,
+            "sRGB",
+            "IEC 61966-2-1:1999",
+            "http://www.color.org",
+            icc_profile,
+            "sRGB2014 (v2)",
+        )
```

Comment:
```suggestion
```
I don't think that really needs to be in the constructor.
I suggest to move this below.